### DISCUSSION TITLE
ssr does not work in node

### DISCUSSION THREAD

**Discussion by bwpark** [nuxt/nuxt#16887] [Questions]
1. How can I check if it works with ssr?
2. As the api is called from the browser when entering the page, ssr does not seem to work.
3. Do I need any other settings besides { ssr: true} in the product?

**Comment by danielroe** [comment]
You can check whether SSR works by inspecting the HTML returned from the server.

```bash
curl http://localhost:3000
```

No settings are required to enable this. If you are experiencing a problem, creating [a minimal reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction) can be of great help in figuring out what's going wrong.
